website/src/components/Edit/EditPage.spec.tsx

```diff
@@ -5,19 +5,25 @@ import { sentenceCase } from 'change-case';
 import { beforeEach, describe, expect, test } from 'vitest';
 
 import { EditPage } from './EditPage.tsx';
-import { defaultReviewData, editableEntry, metadataKey, testAccessToken, testOrganism } from '../../../vitest.setup.ts';
+import { defaultReviewData, editableEntry, metadataDisplayName, testAccessToken, testOrganism } from '../../../vitest.setup.ts';
 import type { UnprocessedMetadataRecord } from '../../types/backend.ts';
+import type { InputField } from '../../types/config.ts';
 import type { ClientConfig } from '../../types/runtimeConfig.ts';
 
 const queryClient = new QueryClient();
 
 const dummyConfig = { backendUrl: 'dummy' } as ClientConfig;
-const inputFields = [
-    {
-        name: 'originalMetaDataField',
-        dispayName: 'Original Meta Data Field',
-    },
-];
+const groupedInputFields = new Map<string, InputField[]>([
+    [
+        'Header',
+        [
+            {
+                name: 'originalMetaDataField',
```

Comment:
```suggestion
                name: metadataKey,
```

maybe? with the import?